packages/react-components/style/trace-explorer.css

```diff
@@ -315,17 +315,34 @@
     font-size: var(--trace-viewer-ui-font-size1);
     min-height: 23px;
     white-space: nowrap;
+    font-weight: bold;
 }
 
-.shortcuts-table i {
-    color: var(--trace-viewer-button-background);
+.shortcuts-icon {
+    color: var(--theia-button-background);
 }
 
 .shortcuts-table-keybinding {
     min-height: 23px;
     vertical-align: middle;
     align-items: center;
     justify-content: flex-end !important;
+    line-height: 10px;
+    display: flex;
+}
+
+.shortcuts-table-keybinding-key {
+    display: inline-block;
+    border-radius: 3px;
+    vertical-align: middle;
+    font-size: 11px;
+    padding: 3px 5px;
+    margin: 0 2px;
+    background-color: var(--theia-keybindingLabel-background);
```

Comment:
CSS variables that start with --theia should be replaced by --trace-viewer. Then, these variables can be mapped to Theia variables in trace-viewer.css. See PR #977.